src/neo/SmartContract/Native/RoleManagement.cs

```diff
@@ -10,12 +10,22 @@
 
 namespace Neo.SmartContract.Native
 {
+    /// <summary>
+    /// A native contract for managing roles in NEO system.
+    /// </summary>
     public sealed class RoleManagement : NativeContract
     {
         internal RoleManagement()
         {
         }
 
+        /// <summary>
+        /// Gets the list of specified role.
```

Comment:
Gets the list of nodes of a specified role.

Reply:
Updated.